Makefile

```diff
@@ -183,6 +188,17 @@ endif
 	@gzcat $(GZ_FILE) | docker-compose run --rm web ./manage.py dbshell
 	@docker-compose run --rm web ./manage.py migrate --no-input
 
+fetchremotedb: $(shell if [ ! -d "./db_cache/" ]; then mkdir ./db_cache/; fi;)
+fetchremotedb:
+ifndef DB_FILE_NAME
+	$(error $(remotedbload_error_msg))
+endif
+	@echo cleaning cache for ${DB_FILE_NAME}
+	@$(shell if [ -f ./db_cache/${DB_FILE_NAME} ]; then rm ./db_cache/${DB_FILE_NAME};fi;)
+	@echo downloading db...
+	@wget -P ./db_cache/ https://s3.amazonaws.com/public-clean-saved-db-states/${DB_FILE_NAME}
+	@echo "DB ${DB_FILE_NAME} is now up to date. You can load it by running 'make dbload GZ_FILE=./db_cache/${DB_FILE_NAME}'."
```

Comment:
Not a blocker, but this makes it always a two step process.  I'd add another target, say load-remote-db that is something like:
```
load-remote_db: fetch-remote-db load-db
```
and then tweak load-db so it uses ${DB_CACHE_DIR}${DB_FILE_NAME} is ${GZ_FILE} is not set.